### PR TITLE
Make User and Todo APIs, Fixed all comments, and Debuging

### PR DESCRIPTION
**Pull request by harmeshverma01**

### CODE REVIEW

File: Todo_app/view/todo_view.py [hunk 1]
```diff
@@ -15,4 +16,22 @@ def add_todo():
     if sno is None or title is None or descriptions is None:
         return jsonify(message="Invalid Request"),500
     
-    return ""
+    return ({'message':'Created SuccessFully'})
+
+
+@todo_bp.route('/todo-list', methods={"GET"})
```
Comment (aayush-kumar-codes) [inline Todo_app/view/todo_view.py:22]: make this endpoint only list

File: user/view/user_view.py [hunk 2]
```diff
@@ -1,19 +1,69 @@
...
 def add_user():
     if not request.json:
         abort(500)
     
-    name = request.json.get("name", None)
-    email = request.json.get("email", None)
-    role = request.json.get("role", None)
-    
-    if name is None or email is None:
+    form = [add_user]
```
Comment (aayush-kumar-codes) [inline user/view/user_view.py:17]: what is the purpose of this line?

File: user/view/user_view.py [hunk 3]
```diff
@@ -1,19 +1,69 @@
...
 from flask import request
+from user.schema.user import User
+from app.db import db
+import jwt
+
 
 auth_bp = Blueprint('auth', __name__, url_prefix='/auth')
 
+
 @auth_bp.route('/user', methods={"POST", "GET"})
```
Comment (aayush-kumar-codes) [inline user/view/user_view.py:12]: why does this endpoint allows GET request?

File: user/view/user_view.py [hunk 4]
```diff
@@ -1,19 +1,69 @@
...
+    return make_response('could not verify', 401, {'Authentication': 'login Required'})
+
+
+
+auth_bp.route('/users', methods={"GET"})
+def user_list():
+    if not request.json:
+        abort(500)
+        
+    users = User.query.all()
```
Comment (aayush-kumar-codes) [inline user/view/user_view.py:58]: Please check find method for it.

File: Todo_app/view/todo_view.py (hunk 1 above)
Comment (harmeshverma01) [reply Todo_app/view/todo_view.py:22]: fixed

File: user/view/user_view.py (hunk 3 above)
Comment (harmeshverma01) [reply user/view/user_view.py:12]: fixed

File: user/view/user_view.py (hunk 4 above)
Comment (harmeshverma01) [reply user/view/user_view.py:58]: ok

File: user/view/user_view.py (hunk 2 above)
Comment (harmeshverma01) [reply user/view/user_view.py:17]: fixed

**Pull request merged**